docs/docs/agents/agents.md

```diff
@@ -7,13 +7,21 @@ hide:
   - tags
 ---
 
-# Agent quickstart
+# LangGraph quickstart
 
 This guide shows you how to set up and use LangGraph's **prebuilt**, **reusable** components, which are designed to help you construct agentic systems quickly and reliably.
 
-## 1. Create an agent
+## 1. Install LangGraph
 
-To create an agent, use [`create_react_agent`][langgraph.prebuilt.chat_agent_executor.create_react_agent]:
+If you haven't already, install LangGraph:
+
+```
+pip install -U langgraph
```

Comment:
```suggestion
pip install -U langgraph "langchain[anthropic]"
```

Reply:
Should we add comment about why langchain[anthropic] is required?

Reply:
yes